core/os/device/remotessh/commands.go

```diff
@@ -104,7 +105,11 @@ func (t sshShellTarget) Start(cmd shell.Cmd) (shell.Process, error) {
 
 	for _, e := range cmd.Environment.Keys() {
 		if e != "" {
-			prefix = prefix + e + "=" + cmd.Environment.Get(e) + " "
+			val := cmd.Environment.Get(e)
+			if strings.ContainsAny(val, " \t\n") {
+				val = "\"" + val + "\""
```

Comment:
Would it be better to always quote? Not sure if `val` will ever contain quotes of its own, but if it could, perhaps use `text.Quote()`?

Reply:
Sounds good I can do that.